docs/install/full-install.rst

```diff
@@ -0,0 +1,14 @@
+=================
+Full Installation
+=================
+
+For a full Flocker installation, including the Flocker plugin for Docker, the following manual instructions take you through everything you need to do:
+
+.. XXX this introduction could be improved with an image. See FLOC-2077
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
I agree that this should be in the style guide, but the answer is - it depends on how many sub-topics are in a specific list. 
In this case I've chosen 2, as I want the subtopics to be displayed (in other situations I might not)